### DISCUSSION TITLE
Using GitHub to make comics or anything outside the coding universe. Is it OK?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35747] [New to GitHub] [answered]
Hello, everyone!

I recently started using GitHub to keep track of my Unity projects and so far I'm enjoying it quite a lot!
So much, in fact, that I was wondering wether it is possible to use it to host and manage other types of projects that involve very little to no coding at all.

Would I be able to commit episodes of a webcomic in a private repo to be able to work on them on multiple devices while storing them online, or would that go against the purpose of the platform?

Thank you!

**Comment by MitkoVtori** [comment, marked as answer]
There's no problem! It would be cool to see it also 😉
